src/SecurityChecker/Command/SecurityCheckerCommand.php

```diff
@@ -1,12 +1,134 @@
 <?php
-declare(strict_types=1);
+declare(strict_types = 1);
```

Comment:
@dereuromark is it ok for us to use the `strict_types` directive here?

Reply:
If it works, sure.